charts/wbaas-backup/templates/secret-backup-openssl-key.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: backup-openssl-key
+  namespace: {{ .Release.Namespace }}
+type: Opaque
+data:
+  key: {{ .Values.backupKey }}
```

Comment:
We probably want to think of some way to avoid having to keep this in plaintext in a values file